CONTRIBUTING.md

```diff
@@ -3,29 +3,24 @@ the process of getting changes merged in and released.
 
 1. [Setting up Marionette locally](#setting-up-marionette-locally)
 2. [Reporting a bug](#reporting-a-bug)
-  1. [When you have a patch](#when-you-have-a-patch)
-  2. [When you don't have a bug fix](#when-you-dont-have-a-bug-fix)
-3. [Running Tests](#running-tests)
-4. [Updating docs](#updating-docs)
+3. [Submitting patches and fixes](#submitting-patches-and-fixes)
+4. [Running Tests](#running-tests)
 
```

Comment:
Some titles are missed:
 - `When you don't have a bug fix`
 - `Determining your branch`
 - `Submitting a Great Patch`
 - `Solving Issues`
 - `Coding Guidelines`
 - `How we Approve Pull Requests`

Maybe the were not added in purpose.